cranelift-codegen/meta/src/cdsl/instructions.rs

```diff
@@ -430,13 +430,15 @@ impl From<Immediate> for BindParameter {
 pub enum Immediate {
```

Comment:
Can you also remove the dead_code allowance above (and mention in the commit message that this closes this issue)

Reply:
Ok, I'll pull out the Uint8 and Uint128 for now then since they are still unused and leave a comment.